build.gradle

```diff
@@ -0,0 +1,31 @@
+buildscript {
+  repositories {
+    mavenCentral()
+  }
+  dependencies {
+    classpath 'com.android.tools.build:gradle:1.3.1'
+  }
+}
+
+allprojects {
+  repositories {
+    mavenCentral()
+  }
+}
+
+ext {
+  minSdkVersion = 8
+  targetSdkVersion = 23
+  compileSdkVersion = 23
+  buildToolsVersion = '23.0.0'
+  sourceCompatibilityVersion = JavaVersion.VERSION_1_7
+  targetCompatibilityVersion = JavaVersion.VERSION_1_7
+}
+
+ext.deps = [
+    // Test dependencies
+    junit      : "junit:junit:4.10",
+    festandroid: "com.squareup:fest-android:1.0.7",
+    robolectric: "org.robolectric:robolectric:2.2",
+    intellijannotations: "com.intellij:annotations:12.0"
```

Comment:
we use single-quote strings for static strings and double-quote strings only when there's a template replacement (e.g., `"Name: $name"`)